select.go

```diff
@@ -16,6 +16,8 @@ type SelectData struct {
 	PermanentFlags []Flag
 	// Number of messages in this mailbox (aka. "EXISTS")
 	NumMessages uint32
+	// Number of recent messages in this mailbox ("RECENT") (IMAP4rev1 only).
```

Comment:
Could we mention "Obsolete, IMAP4rev1 only" to make it clear that it's not great to use it even on IMAP4rev1?

Reply:
Done.